### ISSUE TITLE
Antialiasing + Unknown Icon for Credits

### ISSUE THREAD

**Issue by DMMaster636** [ShadowMario/FNF-PsychEngine#10959]
### What feature do you want to get added on the **base** engine?

Basically be able to have pixel credit icons and instead of having the stupid flixel thing logo (image below) have a proper unknown credit icon
![image](https://user-images.githubusercontent.com/86197998/197358648-dbdccbff-4f7f-4278-bf75-8157bf579801.png)

### To test your sight, and reliability, please select the option of what should NOT be requested.

Stage Editor, 6K+ support, and winning icons.

**Comment by CaptainBaldi** [comment]
ehhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhh
dont know what you mean by pixel credit icons since you can just make the icon itself pixel
for unknown icon just make one urself and if ur adding credits with no icon just use the unknown one you made

tldr these enhancements suck

**Comment by frantastic24** [comment]
> dont know what you mean by pixel credit icons since you can just make the icon itself pixel

Antialiasing makes pixelated graphics blurry, you can't disable it for credits

It'd be nice to have a default credit icon

**Comment by DMMaster636** [comment]
> ehhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhh dont know what you mean by pixel credit icons since you can just make the icon itself pixel for unknown icon just make one urself and if ur adding credits with no icon just use the unknown one you made
> 
> tldr these enhancements suck

what frantastic24 said
then you could say the same for modpacks if they don't have an icon
also, these enhancements are far from bad, since they fix some graphical things

**Comment by BeastlyGhost** [comment]
https://github.com/ShadowMario/FNF-PsychEngine/pull/10944 solves the lack of unknown icons, as for antialiasing, just turn it off on code, it's easier

**Comment by ShadowMario** [comment]
Done on experimental branch, thank you for the suggestion
